### ISSUE TITLE
HTML issues, add accessibility linter

### ISSUE THREAD

**Issue by joepio** [atomicdata-dev/atomic-data-browser#204]
Some buttons are missing text, some forms have no submit. Add a11y linting.

- https://www.npmjs.com/package/eslint-plugin-jsx-a11y
- other options?

**Comment by joepio** [comment]
Done! And run Axe DevTools to fix other issues.
